hoomd/CMakeLists.txt

```diff
@@ -352,6 +350,9 @@ install(FILES ${_hoomd_headers}
 
 # subdirectories that are not components
 add_subdirectory(filter)
+add_subdirectory(custom)
```

Comment:
Now CMake is giving strange errors related to these subdirectories. I'm not sure why.
```
CMake Error at hoomd/CMakeLists.txt:429 (add_subdirectory):
  The binary directory

    /__w/1/b/hoomd/analyze

  is already used to build a source directory.  It cannot be used to build
  source directory

    /__w/1/s/hoomd/analyze

  Specify a unique binary directory name.


CMake Error at hoomd/CMakeLists.txt:429 (add_subdirectory):
  The binary directory

    /__w/1/b/hoomd/custom

  is already used to build a source directory.  It cannot be used to build
  source directory

    /__w/1/s/hoomd/custom

  Specify a unique binary directory name.


CMake Error at hoomd/CMakeLists.txt:429 (add_subdirectory):
  The binary directory

    /__w/1/b/hoomd/update

  is already used to build a source directory.  It cannot be used to build
  source directory

    /__w/1/s/hoomd/update

  Specify a unique binary directory name
```

Reply:
Fixes as per our discussion